### PR TITLE
update readme to reflect status of this plugin and openshift client p…

### PR DESCRIPTION
**Pull request by gabemontero**
…lugin

Fixes https://github.com/openshift/jenkins-plugin/issues/157

@openshift/sig-developer-experience fyi / ptal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,15 @@ is already provided and can be invoked from your Jenkins jobs, or if you are not
...
 
-To that end, the new experimental [OpenShift Client Plugin for Jenkins](https://github.com/openshift/jenkins-client-plugin) is now offered as a technical preview and is included in the OpenShift Jenkins images on Centos (docker.io/openshift/jenkins-1-centos7:latest and docker.io/openshift/jenkins-2-centos7:latest). The plugin is also available from the Jenkins Update Center.  Envisioned as the long term direction for OpenShift integration from Jenkins jobs, among the features provided are:
+To that end, starting with the 3.7 release of OpenShift, the [OpenShift Client Plugin for Jenkins](https://github.com/openshift/jenkins-client-plugin) is now at GA status, fully supported, and is included in the OpenShift Jenkins images. The plugin is also available from the Jenkins Update Center.  Envisioned as the long term direction for OpenShift integration from Jenkins jobs, among the features provided are:
 
 - A Fluent sytled syntax for use in Jenkins Pipelines
```
Comment (bparees) [inline README.md]: styled

**Comment by bparees** [comment]
one nit unrelated to these changes and lgtm.

File: README.md (hunk 1 above)
Comment (gabemontero) [reply README.md]: updated / thx

**Comment by gabemontero** [comment]
[merge]

**Comment by gabemontero** [comment]
install origin failed:  `Create OpenShift certificates for master hosts.[localhost] openshift_master_facts : Set Default scheduler predicates and priorities`

just going to merge manually

**Pull request merged**